x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -226,7 +197,23 @@ export const benchmarksInputSchema = rt.object({
    *  Sortable fields: id, name, policy_id, namespace, updated_at, updated_by, created_at, created_by,
    *  package.name,  package.title, package.version
    */
-  sort_field: rt.maybe(rt.string()),
+  sort_field: rt.maybe(
+    rt.oneOf(
+      [
+        rt.literal('id'),
+        rt.literal('name'),
+        rt.literal('policy_id'),
+        rt.literal('namespace'),
+        rt.literal('updated_at'),
+        rt.literal('updated_by'),
+        rt.literal('created_at'),
+        rt.literal('created_by'),
+        rt.literal('package.name'),
+        rt.literal('package.title'),
+      ],
+      { defaultValue: 'name' }
+    )
+  ),
```

Comment:
these are the current table column headers: 
![Screen Shot 2022-03-14 at 10 55 41](https://user-images.githubusercontent.com/20814186/158138021-158b044b-0ee4-4402-896d-09f7068ac494.png)

their corresponding keys are:  `package_policy.name`, `package_policy.package.title`, `agent_policy.name`, `agent_policy.agents`, `package_policy.created_by`, `package_policy.created_at` and `rules` which is currently missing from BE. 

given that these are the only sort options from UI, i think we should only accept those. as in -change the `sort_field` and the field mapping accordingly. @CohenIdo ?